lib/diagnostic.rb

```diff
@@ -108,10 +108,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = { :favorite_number => 10, :first_name => "Joe"}'
```

Comment:
This is right but we can shorten it even further like so:

`'person = {first_name: "Lauren", favorite_number: 7}'`